packages/deno/src/generators/application/generator.ts

```diff
@@ -128,7 +145,7 @@ export default async function (
 ) {
   const normalizedOptions = normalizeOptions(tree, options);
 
-  initDeno(tree);
+  await initDeno(tree);
   addProjectConfig(tree, normalizedOptions);
   addFiles(tree, normalizedOptions);
   addPathToDenoSettings(tree, normalizedOptions.projectRoot);
```

Comment:
for this function, when using a standalone project, we more than likely don't need to add the path and should just enable deno instead?

we can follow up with this once we experiment more, this is really only for vscode editor support.

Reply:
True, we can test it out and fix later if needed.